### DISCUSSION TITLE
Problems with dual monitor and NVIDIA drivers

### DISCUSSION THREAD

**Discussion by raffaelemancuso** [hyprwm/Hyprland#1294] [Q&A]
After seeing [this commit](https://github.com/hyprwm/hyprland-wiki/commit/21ad477df7f25b485814b42b71e56e3e2dc690b7) on the wiki, and having version 525.60.11 of the NVIDIA drivers,

1. I removed
```
export WLR_DRM_DEVICES=/dev/dri/card0
```
from the hyprland wrapper

2. I replaced:
```
monitor=HDMI-A-1,preferred,auto,1
monitor=eDP-1,disable
```

with 
```
monitor=HDMI-A-1,preferred,auto,1
monitor=eDP-1,preferred,auto,1
```

in the config file.

Dual monitor DOES work.

But there are the following problems:

1. Firefox crashes at start-up (very very bad for me)
2. Mirroring doesn't work. If I use:
```
monitor=eDP-1,preferred,auto,1
monitor=HDMI-A-1,preferred,auto,1,mirror,eDP-1
```
hyprland will crash at startup

@taylor85345 do you experience any of these?

**Comment by raffaelemancuso** [comment]
Uhm if I leave out `GBM_BACKEND` from the wrapper nor Firefox nor Thunderbird crash anymore, but then screensharing doesn't work

**Comment by taylor85345** [comment]
I've seen both of these issues reported on the Discord, but haven't encountered them myself since I don't generally use Firefox or mirror displays. I'm also fairly certain that I don't have the `GBM_BACKEND` variable set anymore, but screen sharing still works for me. Sadly I can't test anything right now as my laptop is out for repairs at the moment.
